### PR TITLE
Pods 1999 Create HUB View

### PR DESCRIPTION
**Pull request by grantrobson**

### CODE REVIEW

File: app/viewmodels/JourneyTaskList.scala [hunk 1]
```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
+/*
```
Comment (suryabalasingh) [inline app/viewmodels/JourneyTaskList.scala]: multiple licence

File: test/controllers/register/SchemeTaskListControllerSpec.scala [hunk 2]
```diff
@@ -50,7 +53,7 @@ class SchemeTaskListControllerSpec extends ControllerSpecBase {
       val result = controller().onPageLoad()(fakeRequest)
 
       status(result) mustBe OK
-      contentAsString(result) mustBe viewAsString()
+      contentAsString(result).contains(messages("messages__schemeTaskList__heading")) mustBe true
```
[the remark this answers is not in the document]
Comment (abhijeetardale) [reply test/controllers/register/SchemeTaskListControllerSpec.scala]: fixed

File: app/views/schemeTaskList.scala.html [hunk 3]
```diff
@@ -29,4 +48,137 @@
...
+
+    <ol class="task-list">
+        <li>
+            <h2 id="section-about-header" class="heading-large">
+                @messages("messages__schemeTaskList__sectionAbout_header")
+            </h2>
+            <ul class="task-list-items">
+                <li class="task-list-item">
+                    <a id= "section-about-link" href="@jl.about.link.target" aria-describedby="section-about-status">
+                        @messages(jl.about.link.text)
```
[the remark this answers is not in the document]
Comment (abhijeetardale) [reply app/views/schemeTaskList.scala.html]: fixed

File: app/viewmodels/JourneyTaskList.scala (hunk 1 above)
Comment (abhijeetardale) [reply app/viewmodels/JourneyTaskList.scala]: fixed

**Pull request merged**